core/src/proxy/auth.rs

```diff
@@ -1,4 +1,6 @@
+use tonic::Code;
```

Comment:
cargo fmt?

Reply:
weird, i have it set to run fmt on save, but had to manually run to get it to fix this.